leftist.ml

```diff
@@ -0,0 +1,58 @@
+(*  Autor kodu: Przemysław Podleśny *)
+(*  Code reviewer: Jakub Nowak      *)
+
+(*  Korzeń kolejki priorytetowej - albo drzewo puste, albo
+    krotka postaci (lewy_syn, priorytet, wysokość, prawy_syn).  *)
+type 'a queue =
+    Empty_q | Node of 'a queue * 'a * int * 'a queue
```

Comment:
Zastanawiam się, czy labowiec nie utnie punktów za użycie krotki zamiast rekordu. Mówił chyba, że nazwy pól rekordów powinny mieć pełne nazwy, nie skrótowe. A krotka to w ogóle nawet nie ma nazw pól.